wallet-new/integration/Functions.hs

```diff
@@ -307,12 +307,14 @@ runAction wc action = do
             -- We choose from the existing wallets AND existing accounts.
             account <-  pickRandomElement localAccounts
             let walletId = accWalletId account
+                accIdx = accIndex account
                 withoutAccount = filter (not . chosenAccount) localAccounts
                 chosenAccount acct =
-                    accWalletId acct == accWalletId account
-                    && accIndex acct ==  accIndex account
-                accIdx = accIndex account
+                    accWalletId acct == walletId
+                    && accIndex acct ==  accIdx
 
+            -- We don't want to delete accounts in genesis wallets
+            walletIdIsNotGenesis walletId
```

Comment:
we shouldn't be able to delete account from genesis wallet (we don't want to loose that money)